### PR TITLE
Speed constant style

### PR DESCRIPTION
**Pull request by justneocity**
I noticed there was a constants folder (client > src > constants) with multiple files to store different constants. So to stay consistent, I also created a file names speed.ts to store the constant for defaultSpeed and stored it in this directory.

One other thing to note is that defaultSpeed is also a constant defined in a file called constants.ts in src>visualiser-src > util and the linked-list-visualiser file is using this one instead. Should I make it so the linked list visualiser used the main speed constant from the constants folder instead.

### CODE REVIEW

File: client/src/constants/speed.ts [hunk 1]
```diff
@@ -0,0 +1 @@
+export const defaultSpeed = 0.6;
```
Comment (jlin2002) [inline client/src/constants/speed.ts:1]: Consider using this constant in client/src/visualiser-src/linked-list-visualiser/initialiser.ts and deleting the one that's in visualiser-src/linked-list-visualiser/util/constants.ts. It's probably better to have one source of truth.

**Review by jlin2002 — CHANGES_REQUESTED** [review]
Just one small thing with duplicated constants

File: client/src/constants/speed.ts (hunk 1 above)
Comment (justneocity) [reply client/src/constants/speed.ts:1]: Thanks Joanna, I've fixed that:)

File: client/src/visualiser-src/linked-list-visualiser/util/constants.ts [hunk 2]
```diff
@@ -56,4 +56,4 @@ export const pathAttributes = {
 };
 
 // Animation attributes
-export const defaultSpeed = 0.6;
+// export const defaultSpeed = 0.6;
```
Comment (jlin2002) [inline client/src/visualiser-src/linked-list-visualiser/util/constants.ts:59]: Delete this line XD

**Review by jlin2002 — APPROVED** [review]

**Review by jlin2002 — CHANGES_REQUESTED** [review]
Delete the commented line

**Pull request closed without merging**